spec.md

```diff
@@ -332,11 +332,22 @@ service Node {
   rpc NodeUnpublishVolume (NodeUnpublishVolumeRequest)
     returns (NodeUnpublishVolumeResponse) {}
 
+  // NodeGetId is being deprecated in favor of NodeGetInfo and will be
+  // removed in CSI 1.0. Existing drivers, however may depend on this
```

Comment:
still missing a comma after `however`. the text should appear this way `, however,`